<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="LineEditDirectory">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectInFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select folder In</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListWidgetFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowInputImageGray">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show input image Gray</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxShowModeGray">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMaxShowGray">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>32</number>
    </property>
    <property name="value">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinShowGray">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-65536</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispGray">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max disp</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinDispGray">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min disp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowInputImagePC">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show input image PC</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispPseudoColor">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max disp</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinDispPseudoColor">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min disp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMaxShowPseudoColor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>32</number>
    </property>
    <property name="value">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinShowPseudoColor">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-65536</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispGradient">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max disp</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinShowGradient">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>300</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-65536</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinDispGradient">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min disp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMaxShowGradient">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowGradient">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Gradient</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowMask">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Mask</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxThresholdOryginalImage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispGradient_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold Im</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxShowConv">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Converted</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMaxShowConv">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>32</number>
    </property>
    <property name="value">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMinShowConv">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-65536</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinDispPseudoColor_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Min disp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxDispPseudoColor_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max disp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
